workflow/r/barebones/v1/amazon.redshiftodbc.ini

```diff
@@ -0,0 +1,9 @@
+[Driver]
+## - Note that this default DriverManagerEncoding of UTF-32 is for iODBC.
+DriverManagerEncoding=UTF-16
+ErrorMessagesPath=/opt/amazon/redshiftodbc/ErrorMessages
+LogPath=[LogPath]
+SwapFilePath=/tmp
+
+#   iODBC
```

Comment:
Hey @saurabh0402 - This still says iodbc 😞 . Also The above message in Note says `Note that this default DriverManagerEncoding of UTF-32 is for iODBC` while we are using UTF-16. Also the path for `ODBCInstLib ` is still using `iodbc` Can we pl change it to use `unixodbc` ?

Reply:
Updated.